CRM/Core/I18n/Schema.php

```diff
@@ -207,6 +207,7 @@ public static function makeSinglelingualTable(
   public static function addLocale($locale, $source) {
     // get the current supported locales
     $domain = new CRM_Core_DAO_Domain();
+    $domain->id = CRM_Core_Config::domainID();
```

Comment:
@colemanw would we rather be adding the locale across all domains?

Reply:
@seamuslee001 I agree with you but I think that's outside the scope of this PR, as it was already only targeting one domain; at least now it will target the *correct* one.